README.md

```diff
@@ -30,18 +29,20 @@ $ mvn clean package
 
 5.  Install this file via the Dashboard > Package Manager.
 
-Find an area of the documentation that needs to be improved? Please raise an [issue](https://github.com/eXist-db/documentation/issues) or submit a [pull request](https://github.com/eXist-db/documentation/pulls)!
+Find an area of the documentation that needs to be improved? Please raise an [issue](https://github.com/eXist-db/documentation/issues) or better yet submit a [pull request](https://github.com/eXist-db/documentation/pulls)!
 
-Our test-suite performs a validation check on all articles in the `/data` directory when you open a pull request. You can speed up the review process by validating your edits locally before submitting a pull request.
+Our test-suite performs a validation check on all articles when you open a pull request. You can speed up the review process by using `mvn validate` locally before submitting a pull request.
 
 
 ## Building a Release from source
 
 1.  Merge outstanding and reviewed PRs.
 
 2.  Set version numbers in `pom.xml`.
```

Comment:
You should not set the version numbers manually. This is done and prompted for by the mvn release plugin, when you run `mvn release-prepare`

Reply:
does it also prompt for processor dependency?

Reply:
Ah maybe we are talking about different things. I mean that you should not set the Maven project version manually in the `pom.xml`. You are right that they will need to set `exist.version` in the pom.xml